app/views/layouts/application.html.haml

```diff
@@ -40,3 +40,5 @@
     #flash-container= flash_messages
 
     %a.entypo-chevron-up.back-to-top#back-to-top{title: t("layouts.application.back_to_top"), href: "#"}
+
+    = render "report/report_modal"
```

Comment:
The modal markup is needed on all page where there are posts, so streams and SPV. Is there a better place I can put it than in the global layout itself?

Reply:
I don't know if you didn't see my previous comment here or if there is another reason, but you didn't change or comment anything here.